modules/ob-about-buildah.adoc

```diff
@@ -1,10 +1,37 @@
 // This module is included in the following assembly:
 //
-// * installing/installing-sample-build-strategies.adoc
+// * about/build-strategies.adoc
 
-:_content-type: CONCEPT
+:_content-type: REFERENCE
```

Comment:
```suggestion
:_mod-docs-content-type: REFERENCE
```

Reply:
As part of our adoption of the IBM tool chain (WFM), we are being asked to use `:_mod-docs-content-type:`. Seeing as you are here, please update.

Reply:
@mjpytlak, I incorporated this suggestion globally and updated content type to  :_mod-docs-content-type: in all modules of this project.